release.sh

```diff
@@ -53,9 +56,9 @@ else
     git push origin "v$VERSION" || true
 
     bundle exec rake build
-    gem push "$CURDIR/pkg/percy-common-$VERSION.gem"
+    gem push "$CURDIR/pkg/percy-common-"*.gem
```

Comment:
leave this explicit? it's quite weird to publish everything from a directory rather than the specific version that was built?

Reply:
This line was modified to handle rubygems' strange mangling of pre-release semver version strings. Your comments have been addressed in ffd6faf, which now transmogrifies pre-release strings to better accommodate [this behavior](https://github.com/rubygems/rubygems/blob/e2a4895aab3225652d5946dc34623fb4177a2678/lib/rubygems/version.rb#L221).

Reply:
Blast from the past, but TIL "transmogrify" is a word